transformer_sidecar/src/transformer_sidecar/transformer_stats/aod_stats.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2022, IRIS-HEP
```

Comment:
For now it will require a new class here and a reference to it in the transformer's `transformer_capabilities.json`
The next time we need to do this, we can burden that issue with generalizing this. Again, this PR is already way too ambitious

Reply:
This sounds ok for now - but perhaps a note could be added somewhere in the docs for future people so they don't get too lost trying to figure this out?